File: other/tokilearning/Latihan/function.cpp
#include <stdio.h>

using namespace std;

int fact(int a);

int main(){
    int n;
    
    scanf("%d",&n);
    if (n<0 || n>10) printf("ditolak\n");
       else printf("%d\n",fact(n));
    
    return 0;
}

int fact(int a){
    if (a==0) return 1;
    return a*fact(a-1);
}

File: other/tokilearning/Latihan/function.pas
var
  n:integer;

function fact(x:integer):longint;
  begin
    if x=0 then exit(1)
      else exit(x*fact(x-1));
  end;

begin
  readln(n);
  if (n<0) or (n>10) then writeln('ditolak')
    else writeln(fact(n));
end.
